docs/dev/replay-exec.md

```diff
@@ -0,0 +1,82 @@
+This doc describes how to dump an Exec runtime meta and column batch data to a directory,
+and replay the Exec with the dumped meta and data. When encountering a perf issue using this tool
+can dump the runtime meta/data and then replay. It will spend less time when collecting NSYS and 
+NCU information when replaying the dumped runtime meta/data.
+
+# Dump a Exec meta and a column batch data
+
+## compile Spark-Rapids jar
```

Comment:
seems unnessary to me, point to regular build docs so it doesn't get out of date.  If there is something specific needed from this then point it out specifically ... ie do you need the allowConventionalDistJar?    I think your enable dump assumption of the rapids accelerator being enabled covers this.

Reply:
> do you need the allowConventionalDistJar? 

Yes,  when doing replay, should specify a Main class, without `allowConventionalDistJar` compile, it will report:
Error: Failed to load class `org.apache.spark.sql.rapids.test.ProjectExecReplayer`.

```
nsys profile $SPARK_HOME/bin/spark-submit \
  --class org.apache.spark.sql.rapids.test.ProjectExecReplayer \
  --conf spark.rapids.sql.explain=ALL \
  --master local[*] \
  --jars ${PLUGIN_JAR} \
  ${PLUGIN_JAR} /home/chongg/test 265335935
```

Reply:
Updated doc.